framework/build/enyo-build.js

```diff
@@ -7611,7 +7604,7 @@ this.$.view.callBrowserAdapter(a, b);
 webviewClick: function(a, b, c) {
 c && (c.element == "SELECT" ? this._selectRect = c.bounds : this._selectRect = null, this.doClick(b, c));
 }
-}), window.PalmSystem || (enyo.WebView = enyo.Iframe);
+}), !window.PalmSystem && !enyo.args.haveBrowserAdapter && (enyo.WebView = enyo.Iframe);
```

Comment:
Here is some change too

Reply:
This corresponds to fix change: https://github.com/webOS-ports/enyo-1.0/commit/563aae1bed99055d988f81380ddf2e7bd25ebb41